app/src/main/java/com/ayush/weatherapp/utils/LocationUtils.java

```diff
@@ -0,0 +1,37 @@
+package com.ayush.weatherapp.utils;
+
+import android.content.Context;
+import android.content.Intent;
+import android.location.LocationManager;
+import android.provider.Settings;
+import android.support.v7.app.AlertDialog;
+import com.ayush.weatherapp.R;
+
+public final class LocationUtils {
+
+  private LocationUtils() {
+  }
+
+  //Reference: https://stackoverflow.com/questions/10311834/how-to-check-if-location-services-are-enabled
+  public static void checkLocationServices(Context context) {
```

Comment:
The dialog could be separated into it's own function and created in the activity.

Reply:
:ok: